doto-frontend/src/components/pages/Calendar/Calendar.js

```diff
@@ -62,7 +62,7 @@ const Calendar = () => {
         };
         fetchUserInfo();
         fetchTasks();
-    }, []);
+    }, [setTheme]);
```

Comment:
This is a better solution. Otherwise you get the behaviour I experienced this morning
![2020-04-09-075925_832x1390_scrot](https://user-images.githubusercontent.com/18223858/78862840-5ba87c00-7a8c-11ea-83b4-6cb7e341b40e.png)

Reply:
Thanks @FocalChord for pointing it out.